src/compiler/test-file/formats/raw/prepare-command-object.js

```diff
@@ -0,0 +1,23 @@
+// NOTE: TestCafe Studio adds additional fields to the command object in RAW tests.
```

Comment:
I propose using the more specific name for this function - `removeStudioRelatedProperties`. Also, this function can be moved to the `RawTestFileCompiler` class. It's simple and doesn't used in other places.